libmscore/layoutlinear.cpp

```diff
@@ -83,6 +83,9 @@ static void processLines(System* system, std::vector<Spanner*> lines, bool align
       if (!firstMeasure())
             return;
 
+      for (MeasureBase* mb = first(); mb; mb = mb->next())
+            mb->setSystem(nullptr);
+
```

Comment:
For some reason, git thinks this change is in `processLines()`, but it is actually in `Score::resetSystems()`.

Reply:
if each measure's _system_ is not set to _nullptr_ (or a valid system), then bad things will happen during `Score::getNextMeasure()`.